console/account/how-to/resolve-abuse.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to resolve abuse tickets and cases
+  description: This page explains how to resolve abuse tickets.
+content:
+  h1: How to resolve abuse tickets and cases
+  paragraph: This page explains how to resolve abuse tickets.
+tags: resolve abuse complaint ticket scaleway
+dates:
+  validation: 2024-02-19
+  posted: 2024-02-19
+categories:
+  - account
+---
+
+Abuse tickets are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuse reports for the same issue on the same resource within a given timeframe are grouped by case.
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the locking of their services.
+However, Scaleway reserves the right to immediately suspend the infringing services if it is deemed necessary and proportionate to keep Scaleway’s network safe or to abide by legal requirements.
+
+If you encounter any issue with the resolution of your abuse tickets, please contact the abuse team by opening a [support ticket](https://console.scaleway.com/support/tickets/create) directly from your console.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+</Message>
+
+1. Click **Abuse** in the bottom left corner of the Scaleway console. A list of your open cases displays. You can also view the closed cases by clicking on “Closed cases”.
+2. Click the **case ID** to view the specifics of the complaint.
+3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue. 
+    <Message type="tip">
+      If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](https://www.scaleway.com/en/docs/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.
+    </Message>
+    In case you receive an abuse ticket for hosting illegal content (including copyright), please delete the content from Scaleway infrastructure promptly.
```

Comment:
```suggestion
    In case you receive an abuse report for hosting illegal content (including copyright), please delete the content from Scaleway infrastructure promptly.
```